apps/src/sites/studio/pages/shared/_header_progress.js

```diff
@@ -14,4 +15,42 @@ if (container.length) {
     </Provider>,
     container[0]
   );
+
+  if (true) {
+    const usersIcon = document.createElement('i');
+    usersIcon.className = 'fa fa-fw fa-users';
+    const userCount = document.createElement('span');
+    userCount.textContent = '0';
+    const userCountDiv = document.createElement('div');
+    userCountDiv.appendChild(usersIcon);
+    userCountDiv.appendChild(userCount);
+    userCountDiv.style.marginRight = '1em';
+    container[0].parentNode.insertBefore(userCountDiv, container[0]);
+
+    const pubSub = createPubSub(getScriptData('pubSub'));
+    const channel = pubSub.subscribe(`presence-test`);
```

Comment:
We're using a [presence channel](https://pusher.com/docs/channels/using_channels/presence-channels) (specified by the `presence-` prefix on the channel name) to get access to the `member_added` and `member_removed` events so we can count people on the channel.  See also [this user list quick-start](https://pusher.com/docs/channels/getting_started/javascript-realtime-user-list) tutorial.

Note: NetSim doesn't do this, even though it probably could/should; I don't remember if presence channels weren't available five years ago or we just didn't know about them, but we manage our own lobby with a Redis store instead of using this feature.